app/controllers/users/backup_code_setup_controller.rb

```diff
@@ -121,7 +139,7 @@ def generator
     end
 
     def authorize_backup_code_disable
-      return if MfaPolicy.new(current_user).multiple_factors_enabled?
+      return if MfaPolicy.new(current_user).multiple_factors_enabled? || in_account_creation_flow?
```

Comment:
I think we'll want to avoid `in_account_creation_flow?` across the board, similar to in d96490bd7e94dad7b78f7afc2f7f9ee3d50bf580.